plugins/installer.plugin

```diff
@@ -28,18 +28,41 @@ finalize_deployment_installer()
   echo " * creating installer initrd"
 
   KERNEL_ARCH="$ARCH"
-
   if [ "$ARCH" = "i386" ]; then
     # The kernel architecture of i386 (32 bit) kernels is called 686
     KERNEL_ARCH="686"
   fi
+
+  if [ "$INSTALLER_BOOTTYPE" = "efi" ]; then
+    INSTALLER_EXTRA_PACKAGES="$INSTALLER_EXTRA_PACKAGES grub-efi-$ARCH-bin dosfstools"
```

Comment:
Why not install the grub-efi meta package instead of arch specific one?

Reply:
it is not needed for creating the initrd ;)